clusterloader2/testing/load/config.yaml

```diff
@@ -15,22 +16,39 @@
 {{$bigRCsPerNamespace := DivideInt $bigRCs $namespaces}}
 {{$mediumRCsPerNamespace := DivideInt $mediumRCs $namespaces}}
 {{$smallRCsPerNamespace := DivideInt $smallRCs $namespaces}}
-{{$totalPodsRounded := AddInt (MultiplyInt $bigRCs $BIG_GROUP_SIZE) (MultiplyInt $mediumRCs $MEDIUM_GROUP_SIZE) (MultiplyInt $smallRCs $SMALL_GROUP_SIZE)}}
+{{$totalPods := AddInt (MultiplyInt $bigRCs $BIG_GROUP_SIZE) (MultiplyInt $mediumRCs $MEDIUM_GROUP_SIZE) (MultiplyInt $smallRCs $SMALL_GROUP_SIZE)}}
 # saturationQPS - qps for RC operations should be an equivalent to 10 [pods/s].
-{{$saturationQPS := DivideFloat (AddInt $bigRCs $mediumRCs $smallRCs) (DivideFloat $totalPodsRounded 10)}}
+{{$bigSaturationQPS := DivideFloat $bigRCs (DivideFloat $totalPods 10)}}
+{{$mediumSaturationQPS := DivideFloat $mediumRCs (DivideFloat $totalPods 10)}}
+{{$smallSaturationQPS := DivideFloat $smallRCs (DivideFloat $totalPods 10)}}
```

Comment:
>> Why do you say it's a bad idea?

> It's not a bad idea. It will just require reimplementing all tuning sets and how the phases are handled and this will take time.

I just looked into it and I didn't realize that tuning sets are implemented based on tickers.
I think it's really limiting implementation.

In fact that they should have access to the operations that they are supposed to run.
[So I think that at the very high-level the api for tuning-set implementatio should be:
func (ts *TuningSet) Execute(actions) {}
rather than implemented based on ticker.

That is much more powerful and gives more freedom in defining their types (as an example what I suggested with randomized load).

I understand that it will take time to implement it, but I really think we should do this (I wish I was around when this was implemented to point that at that time).

Reply:
#259 - Rework of TuningSets.

Reply:
So to copy my initial comment on that:

> TBH, I don't think it's exaclty what we want.
I think we should go a different direction to make the test much simpler.

> What we know is that we would like to generate average 10pods/s throughput.
So I would suggest introducing a different TuningSet type like:
RandomizedLoad
but where you pass total time over which you spread operations, not the QPS.

>That would make it waaaay simpler, because all you will need is one tuningset, not one tunningset per type.

that should be relatively simple to do now

Reply:
#262 - Time limited tuning sets.

Reply:
OK - this is merged. So we can use it here.

Reply:
Changes applied.